VideoAPI/Video.API/Controllers/ConferenceController.cs

```diff
@@ -491,16 +491,31 @@ private async Task DeleteAudioRecordingApplication(Guid conferenceId)
 
         private async Task EnsureAudioFileExists(Conference conference)
         {
-            var filePath = $"{conference.HearingRefId}.mp4";
             var azureStorageService = _azureStorageServiceFactory.Create(AzureStorageServiceType.Vh);
+            var allBlobs = await GetAllBlobNamesByFilePathPrefix(conference.HearingRefId.ToString(), azureStorageService);
 
-            if (!await azureStorageService.FileExistsAsync(filePath) && conference.ActualStartTime.HasValue)
+            if (!allBlobs.Any() && conference.ActualStartTime.HasValue)
             {
                 var msg = $"Audio recording file not found for hearing: {conference.HearingRefId}";
                 throw new AudioPlatformFileNotFoundException(msg, HttpStatusCode.NotFound);
             }
         }
 
+        private static async Task<IEnumerable<string>> GetAllBlobNamesByFilePathPrefix(string filePathPrefix, IAzureStorageService azureStorageService, string fileExtension = ".mp4")
```

Comment:
not sure this belongs in a controller?

Reply:
I think currently I could not move it to service, as actually it's required bigger refactoring in the another parts.   Need to review the implementation of base class and interface for service as there a some duplications for VH and CVP, I think it better to do in separate task. As it just a fix.